apps/frontend/src/locales/en/translations.json

```
@@ -492,6 +492,8 @@
         }
     },
     "maintenanceMode": {
-        "fastBtc": "Fast BTC is locked"
+        "fastBtc": "Fast BTC is locked",
+        "fullD2": "Sovryn is in maintenance mode.<br/>Follow the social channels linked in the footer for further updates.",
```

```suggestion
        "fullD2": "Sovryn is in maintenance mode. Follow the social channels linked in the footer for further updates.",
```

No HTML in localisation file if possible please